src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private final ColorSupplier color;
+    private final Random random;
+
+    public Lottery(ColorSupplier colorSupplier, Random random) {
+        this.color = colorSupplier;
+        this.random = random;
+    }
+
+    public Ball getRandomBall() {
+        Random random = new Random();

```

Comment:
you already have 'random' declared as a class field. Let's remove this redundant local variable